### ISSUE TITLE
Tag is 0.4.7 but associated file is ANTsR_0.4.8_R_x86_64-pc-linux-gnu_R.tar.gz

### ISSUE THREAD

**Issue by dorianps** [ANTsX/ANTsR#264]
Note as per title.

**Comment by muschellij2** [comment]
Tags don't always match up.  The version in the tarball is the correct one.
John

**Comment by dorianps** [comment]
Aren't release tags placed manually? Why not keeping release version same as package version?

**Comment by muschellij2** [comment]
Yes they are.  As they are done manually, mistakes are made as you can see.
John
